metricbeat/docs/modules_list.asciidoc

```diff
@@ -130,7 +130,7 @@ This file is generated! See scripts/docs_collector.py
 |<<metricbeat-metricset-nats-subscriptions,subscriptions>>   
 |<<metricbeat-module-nginx,Nginx>>     |image:./images/icon-yes.png[Prebuilt dashboards are available]    |  
 .1+| .1+|  |<<metricbeat-metricset-nginx-stubstatus,stubstatus>>   
-|<<metricbeat-module-oracle,Oracle>>  beta[]   |image:./images/icon-no.png[No prebuilt dashboards]    |  
+|<<metricbeat-module-oracle,Oracle>>  beta[]   |image:./images/icon-yes.png[Prebuilt dashboards are available]    |  
```

Comment:
Consider also adding a screenshot to the documentation.

Reply:
Once the dashboard is accepted and merged, hence we are "sure" about its final appereance; I'll do the PR to add the screenshot. Thanks